src/encoded/audit/experiment.py

```diff
@@ -48,6 +48,34 @@
 ]
 
 
+def audit_HiC_restriction_enzyme_in_libaries(value, system, excluded_types):
+    '''
+    Libraries for HiC experiments should use the same restriction enzyme
+    '''
+    if value['assay_term_name'] != 'HiC':
+        return
+    if value['status'] in ['deleted', 'replaced', 'revoked']:
+        return
+    if 'replicates' not in value:
+        return
+    frag_methods = set()
+    using_restriction_enzyme = 0
+    for rep in value['replicates']:
+        if rep.get('status') not in excluded_types and 'library' in rep:
+            lib = rep['library']
+            if lib.get('status') not in excluded_types and \
+                    'fragmentation_method' in lib:
+                if 'restriction' in lib['fragmentation_method']:
+                    using_restriction_enzyme = 1
+                frag_methods.add(lib['fragmentation_method'])
+    if len(frag_methods) > 1 and using_restriction_enzyme > 0:
+        detail = 'Experiment {} '.format(value['@id']) + \
```

Comment:
Don't mix string functions.   I know this pattern is all over our code base so try to fix it when updating.

The detail can be written with one string format function 
```
detail = 'Experiment {} contains libraries with inconsistent restriction enzymes {}'.format(
    value['@id'],
    frag_methods
)
```
or 
```
detail = (
    'Experiment {} contains libraries with '
    'inconsistent restriction enzymes {}'.format(
        value['@id'],
        frag_methods,
    )
)
```